views/account/account-main.ejs

```diff
@@ -3,29 +3,63 @@
-      Et accusantium ducimus maiores ullam tempore ea. general stats: Lorem
-      ipsum dolor sit amet consectetur, adipisicing elit. Non inventore aut
-      nesciunt neque expedita molestias, dolor nisi labore enim laboriosam quos
-      quae aliquam? Et accusantium ducimus maiores ullam tempore ea. general
-      stats: Lorem
+    <div class="info-box-div account-first-row">
+      <div class="personal-info">
+        <p class="personal-info-item">Name:</p>
+        <p class="personal-info-item">Age:</p>
+        <p class="personal-info-item">Location:</p>
+      </div>
+      <p class="bio">bio</p>
+    </div>
+    <div class="info-box-div">
+      <h3 class="highscore-title">High Scores</h3>
+      <div class="grid-container">
+        <div class="grid-item"></div>
+        <div class="grid-item gird-difficulty">
+          <p class="grid-text-full">Very Easy</p>
+          <p class="grid-text-short">V.Easy</p>
+        </div>
+
+        <div class="grid-item gird-difficulty">Easy</div>
+        <div class="grid-item gird-difficulty">Medium</div>
+        <div class="grid-item gird-difficulty">Hard</div>
+        <div class="grid-item gird-difficulty">
+          <p class="grid-text-full">Very Hard</p>
+          <p class="grid-text-short">V.Hard</p>
+        </div>
+        <div class="grid-item grid-length">1</div>
+
+        <div class="grid-item grid-score">score</div>
+        <div class="grid-item grid-score">score</div>
+        <div class="grid-item grid-score">score</div>
+        <div class="grid-item grid-score">score</div>
+        <div class="grid-item grid-score">score</div>
+        <div class="grid-item grid-length">2</div>
+        <div class="grid-item grid-score">score</div>
+        <div class="grid-item grid-score">score</div>
+        <div class="grid-item grid-score">score</div>
+        <div class="grid-item grid-score">score</div>
+        <div class="grid-item grid-score">score</div>
+        <div class="grid-item grid-length">3</div>
+        <div class="grid-item grid-score">score</div>
+        <div class="grid-item grid-score">score</div>
+        <div class="grid-item grid-score">score</div>
+        <div class="grid-item grid-score">score</div>
+        <div class="grid-item grid-score">score</div>
+        <div class="grid-item grid-length">4</div>
+        <div class="grid-item grid-score">score</div>
+        <div class="grid-item grid-score">score</div>
+        <div class="grid-item grid-score">score</div>
+        <div class="grid-item grid-score">score</div>
+        <div class="grid-item grid-score">score</div>
+        <div class="grid-item grid-length">5</div>
+        <div class="grid-item grid-score">score</div>
+        <div class="grid-item grid-score">score</div>
+        <div class="grid-item grid-score">score</div>
+        <div class="grid-item grid-score">score</div>
+        <div class="grid-item grid-score">score</div>
```

Comment:
`grid-item` is not targeted by any css selectors so no point in bloating HTML with unused classes.